proto/topup/v1beta/msg.proto

```diff
@@ -22,9 +22,9 @@ service Msg {
 // GenesisState defines the topup module's genesis state.
 message MsgTopup {
     option (gogoproto.goproto_getters) = false;
-    string from_address = 1;
-    string user = 2;
-    uint64 fee = 3
+    string from_address                = 1;
```

Comment:
We should add gogoproto annotations for multi-word keys. ref https://github.com/cosmos/cosmos-sdk/blob/v0.40.1/proto/cosmos/auth/v1beta1/auth.proto#L45

Reply:
Copy

Reply:
Addressed in another PR